src/utils/daos_hdlr.c

```diff
@@ -1653,7 +1725,8 @@ fs_copy_dir(struct cmd_args_s *ap,
 	    const char *src_path,
 	    const char *dst_path,
 	    uint64_t *num_dirs,
-	    uint64_t *num_files)
+	    uint64_t *num_files,
+	    uint64_t *num_links)
```

Comment:
At this point the arg list is getting quite long, perhaps define a fs_copy_stats struct to avoid such a long list and just pass in one pointer.

Reply:
I just made the change you suggested.

Reply:
I only revised fs_copy with "struct fs_copy_args *copy_args" as arguments. I did not modified fs_copy_file, fs_copy_dir and fs_copy_symlink in the same style since fs_copy_dir is a recursive function with struct fs_copy_args partially updated. The benefit to use struct parameters in these three functions is not large enough since we need to add a lot of "copy_args->" or add multiple local variables.

Reply:
FWIW, this ticket would capture more metrics and would probably add a struct then: https://daosio.atlassian.net/browse/DAOS-9692

Reply:
for fs_copy() I was thinking of a struct with just the num_ entries rather than all the args.  We can work on this in future PRs though.